food-truck-api/src/main/resources/application-dev.properties

```diff
@@ -1,3 +1,3 @@
 spring.datasource.url=jdbc:mysql://localhost:3308/foodtruckdb
 spring.datasource.username=root
-spring.datasource.password=password
```

Comment:
Did this actually work locally? I'm pretty sure we need to keep this for our dev environments.